www_src/pages/login/sign-up.jsx

```diff
@@ -47,6 +48,8 @@ var SignUp = React.createClass({
     {
       name: 'username',
       label: 'Username',
+      type: 'email',
```

Comment:
Trick to avoid keyboard suggestions on Android (which cause a noticeable "glitch" when switching between fields) and not wanted when typing a username.